HCore-Storage/HCore-Storage.csproj

```diff
@@ -7,10 +7,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.15.0" />
-    <PackageReference Include="Google.Apis" Version="1.57.0" />
-    <PackageReference Include="Google.Cloud.Storage.V1" Version="2.5.0" />
-	<PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="17.2.32">
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.19.1" />
+    <PackageReference Include="Google.Apis" Version="1.66.0" />
+    <PackageReference Include="Google.Cloud.Storage.V1" Version="4.8.0" />
```

Comment:
Major version change. Tested?

Reply:
I still remember that there were issues with the signed link not working anymore...

Reply:
Cloud storage and storage blob fully tested